docs/Users-Guide.md

```diff
@@ -63,18 +128,21 @@ tsctl runserver
 #### Adding users
 
 Command:
-```
+
+```shell
```

Comment:
again, these are all **sysadmin** functions, not really end user

Reply:
maybe split the docuemtn up into `admin` and `user` guide

Reply:
done